src/m4a_tables.c

```diff
@@ -261,10 +261,9 @@ const struct PokemonCrySong gPokemonCrySongTemplate =
     0, // block count
     255, // priority
     0, // reverb
-    (struct ToneData *)&voicegroup000,
-    NULL,
-    NULL,
-    0,
+    (struct ToneData *)&voicegroup000, // tone
+    {NULL, NULL}, // part
+    0, // gap
```

Comment:
You can specify the struct member names with `.tone =` etc. if you want to do that, rather than comment their names. Otherwise just the fix is fine here.

Reply:
You're right, that's better. Will do.